### DISCUSSION TITLE
Add meta data

### DISCUSSION THREAD

**Discussion by jonaprieto** [squidfunk/mkdocs-material#5372] [Q&A] [answered]
I couldn't find clear instructions on how to add this kind of metadata (3 months ago and 11 months ago):

![image](https://user-images.githubusercontent.com/1428088/232330117-fa72d1da-05e1-45d4-88cc-38d32f9ce823.png)

**Comment by jonaprieto** [comment]
Also, is there an option to not display this information on some pages?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hi, it's all documented here https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#revisioning
You need to install and enable a plugin. The way the information is displayed is defined in the source-file partial here https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/source-file.html

As for the ability to turn it off for some pages, yes that's possible.
https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/options/#exclude

Also duplicate of https://github.com/squidfunk/mkdocs-material/discussions/5301 ✌️
